docs/install-mlrun-ce/mlrun-ce-overview.md

```diff
@@ -0,0 +1,81 @@
+
+4. **Scalability and Efficiency** - Automatically and elastically scale resources based on demand. This ensures your workloads, whether it is batch or real-time, run efficiently, reducing computation costs. It's particularly useful for resource-intensive tasks like LLM fine-tuning or inference.
+
+5. **MLRun Model Monitoring** – Features a comprehensive model monitoring solution that lets users track their models, compare results and performance metrics, and detect data drift or anomalous behavior. 
+It also supports automated alerts for model exceptions, enabling proactive maintenance and ensuring continued model reliability.
+
+6. **Seamless Integrations** – MLRun CE seamlessly connects with a broad ecosystem of leading open-source tools—including Kubeflow Pipelines (KFP) for workflow orchestration, Spark for large-scale data processing, and Grafana for interactive visualization. Its flexible, open architecture enables you to incorporate your preferred tools and workflows, accelerating adoption and productivity.
+
+## Ecosystem
+<p align="center"><img src="../_static/images/mlrun-ce-diagram.jpg" alt="MLRun CE Ecosystem" width="800"/></p><br>
+
+## Core Components
+* MLRun - https://github.com/mlrun/mlrun
+  - MLRun API
+  - MLRun UI
+  - MLRun DB (MySQL)
+* Nuclio - https://github.com/nuclio/nuclio
+* Jupyter - https://github.com/jupyter/notebook (+MLRun integrated)
+* MPI Operator - https://github.com/kubeflow/mpi-operator
+* MinIO - https://github.com/minio/minio/tree/master/helm/minio
+* Spark Operator - https://github.com/GoogleCloudPlatform/spark-on-k8s-operator
+* Prometheus stack - https://github.com/prometheus-community/helm-charts
+  - Prometheus
+  - Grafana
+* MLRun Model Monitoring - 
+  - Kafka - https://github.com/bitnami/charts/tree/main/bitnami/kafka
+  - TDengine - https://github.com/taosdata/TDengine-Operator/blob/3.0/helm/tdengine/values.yaml
+* KFP Pipelines - https://github.com/kubeflow/pipelines
+
+## **Storage resources**
+When installing the MLRun Community Edition, several storage resources are created:
+
+- **PVs via default configured storage class**: Holds the file system of the stacks pods, including the MySQL database of MLRun, MinIO for artifacts and Pipelines Storage and more. 
+These are not deleted when the stack is uninstalled, which allows upgrading without losing data.
+
+   See also MLRun data store [documentation](https://docs.mlrun.org/en/stable/store/datastore.html)
+
+- **Container Images in the configured docker-registry**: When building and deploying MLRun and Nuclio functions via the MLRun Community Edition, the function images are 
+stored in the given configured docker registry. These images persist in the docker registry and are not deleted.
+
+## Addional References 
+
+- **Documentation:** [MLRun Docs](https://docs.mlrun.org) | [Nuclio Docs](https://docs.nuclio.io/en/latest/index.html)
```

Comment:
- **Documentation:** [MLRun docs](https://docs.mlrun.org) | [Nuclio docs](https://docs.nuclio.io/en/latest/index.html)
- **Quick start:** [MLRun basics tutorial](https://docs.mlrun.org/en/stable/tutorials/01-mlrun-basics.html)
- **Cheat sheet:** [MLRun cheat sheet](https://docs.mlrun.org/en/stable/cheat-sheet.html)
- **Community:** [Join our Slack](https://mlopslive.slack.com) for support and discussions
- **GitHub:** [MLRun Repository](https://github.com/mlrun/mlrun)